scripts/Enumerate-MSBuild.ps1

```diff
@@ -33,7 +33,7 @@ foreach ($instance in $vsInstances)
     $instancePath = $instance.installationPath
     Write-Log "********************" -LogToConsole $False
     Write-Log "Found VS Instance: $instanceName"
-    
```

Comment:
Just realized this file was renamed. Please don't do that. Your name is better but we have links all over the Internet pointing to the old bad name.

Reply:
I see! Didn't think of it as an intrusive change!! Sorry!!!